config/filter.d/postfix-sasl.conf

```diff
@@ -7,7 +7,7 @@ before = common.conf
 
 [Definition]
 
-_daemon = postfix(-\w+)?/(submission/)?smtp(d|s)
+_daemon = postfix(-\w+)?/((?:submission|smtps)/)?smtpd
```

Comment:
Are you sure that daemon prefix like `postfix-.../submission/smtps` resp. `postfix-.../smtps` (both with `s` at end) will never occur anymore?
Because if not, it should be:
```diff
-_daemon = postfix(-\w+)?/(submission/)?smtp(d|s)
+_daemon = postfix(-\w+)?/(?:submission/|smtps/)?smtp[ds]
```
Either way I'll prefer this one, to be really backwards compatible...